index.html

```diff
@@ -130,22 +130,22 @@ <h2 class="refer-link">LET'S CONNECT</h2>
                 </div>
             </div>
         </section>
-        <section id="contact-container">
-            <div id="contact-items">
+        <section id="contactcontainer">
+            <div id="contactitems">
                 <div id="description-contact-box">
                     <h5 class="contact-title-1">Contact me</h5>
                     <p class="contact-description-1">
                         If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.
                     </p>
                 </div>
-                <form id="input-contact-box" action="https://formspree.io/f/xpzbqlab" method="post">
+                <form id="inputcontactbox" action="https://formspree.io/f/xpzbqlab" method="post">
```

Comment:
### OPTIONAL 

- [ ] Good job validating your contact form, currently when I click submit the contact form does not submit please ensure that a user can submit forms after it passes all the set validations 👍🏿